doc/man3/flux_future_set_result.adoc

```diff
@@ -0,0 +1,157 @@
+flux_future_set_result(3)
+=========================
+:doctype: manpage
+
+
+NAME
+----
+flux_future_set_result, flux_future_set_error, flux_future_aux_get, flux_future_aux_set, flux_future_set_flux, flux_future_get_flux - support methods for future wrappers
+
+
+SYNOPSIS
+--------
+ #include <flux/core.h>
+
+ void flux_future_set_result (flux_future_t *f,
+                              void *result, flux_free_f free_fn);
+
+ void flux_future_set_error (flux_future_t *f, int errnum);
+
+ void *flux_future_aux_get (flux_future_t *f, const char *name);
+
+ int flux_future_aux_set (flux_future_t *f, const char *name,
+                          void *aux, flux_free_f destroy);
+
+ void flux_future_set_flux (flux_future_t *f, flux_t *h);
+
+ flux_t *flux_future_get_flux (flux_future_t *f);
+
+
+DESCRIPTION
+-----------
+
+A Flux future represents some activity that may be completed with reactor
+watchers and/or message handlers.  It is intended to be wrapped
+by other classes that perform specific activities, providing a common idiom
+for synchronizing on the result.  Typically such a class provides
+class-specific wrappers for `flux_future_create()` and `flux_future_get()`,
+and implements a class-specific init callback.
+
+`flux_future_create()` is called with an init callback.  The callback sets
```

Comment:
Since the init callback is now tied to the _create() function, I think the documentation about it should be moved there.